tests/test_target/test_binning.py

```diff
@@ -61,3 +61,15 @@ def test_pandas_bins(self):
         visualizer.fit(y)
         visualizer.finalize()
         self.assert_images_similar(visualizer, tol=0.5)
+    
+    def test_quick_method(self):
+        """
+        Test the quick method with producing a valid visualization
+        """
+        data = load_occupancy(return_dataset=True)
+        _, y = data.to_pandas()
+        
+        visualizer = balanced_binning_reference(y, show=False)
+
+        assert isinstance(visualizer, BalancedBinningReference)
+        self.assert_images_similar(visualizer, tol=0.5)
```

Comment:
@Kautumn06 @wagner2010 I notice that the Appveyor Miniconda test is hanging indefinitely on this test. One thing that you could try is *either* 
1. changing line 70 to ` _, y = data.to_numpy()` 
OR 
2. adding the `@pytest.mark.skipif(pd is None, reason="pandas is required")` decorator to the `test_quick_method` function (since if you have the line `_, y = data.to_pandas()`, then the test requires Pandas, which not all testing environments will have)

Reply:
GM @rebeccabilbro @Kautumn06 I'll go for option 2 using the in-line editor.  Thank you both!